src/js/angular/repositories/controllers.js

```diff
@@ -10,9 +10,35 @@ export const getFileName = function(path) {
     return name;
 };
 
+const parseNumberParamsIfNeeded = function (params) {
+    if (params && params.queryTimeout && params.queryLimitResults) {
+        // Parse both parameters properly to number
+        params.queryTimeout.value = parseInt(params.queryTimeout.value);
+        params.queryLimitResults.value = parseInt(params.queryLimitResults.value);
+    }
+}
+
+const getFaFaAngleClass = function () {
```

Comment:
Please, use a more meaningful name for the function, i.e getShaclOptionsClass or something like that.